CMake/SlicerMacroBuildLoadableModule.cmake

```diff
@@ -218,8 +218,10 @@ macro(slicerMacroBuildLoadableModule)
   set_target_properties(${lib_name} PROPERTIES LABELS ${lib_name})
 
   target_link_libraries(${lib_name}
-    ${Slicer_GUI_LIBRARY}
-    ${LOADABLEMODULE_TARGET_LIBRARIES}
+    PUBLIC
+      ${LOADABLEMODULE_TARGET_LIBRARIES}
+    PUBLIC
+      ${Slicer_GUI_LIBRARY}
```

Comment:
This could be simplified to

```
target_link_libraries(${lib_name}
  PUBLIC
    ${LOADABLEMODULE_TARGET_LIBRARIES}
    ${Slicer_GUI_LIBRARY}
    )
```

Any particular reason, you used `PUBLIC` twice ?

Reply:
That particular formulation fails if LOADABLEMODULE_TARGET_LIBRARIES has the PRIVATE keyword within it.

An alternative is to reorder it and put GUI_LIBRARY first.

Reply:
Add this info in a comment to prevent other developers in the future remove the "unnecessary" duplication.